dropwizard-jetty/src/main/java/io/dropwizard/jetty/NonblockingServletHolder.java

```diff
@@ -13,7 +13,12 @@
 /**
  * A {@link ServletHolder} subclass which removes the synchronization around servlet initialization
  * by requiring a pre-initialized servlet holder.
+ *
+ * @deprecated If necessary, use {@link ServletHolder} or {@link org.eclipse.jetty.servlet.FilterHolder} directly.
+ * This class will be removed in Dropwizard 2.1.0.
```

Comment:
maybe remove the `handle()` override to avoid the jetty incompatibility in anything that's using this directly? 
(maybe remove all the overrides and just have this essentially act as an alias for ServletHolder?)

https://github.com/search?q=NonblockingServletHolder&type=code shows some usages that are not just forks of DW, fwiw

Reply:
> maybe remove the handle() override to avoid the jetty incompatibility in anything that's using this directly?

This wouldn't be a problem in Dropwizard 2.0.15 which is being built against Jetty 9.4.33, would it?

Reply:
probably not unless someone tries to run DW 2.0.15 / jetty 9.4.32?  or future breakage in 9.4.34+?  not sure how much work is worth trying to protect against unknowns like that though.